### PR TITLE
Creating Minimum Ontology Registration Request Checklist

### PR DESCRIPTION
**Pull request by matentzn**
This checklist should contain more or less the consensus of #1116. Comments welcome @cmungall @jamesaoverton @wdduncan @nlharris @balhoff

### CODE REVIEW

**Review by nlharris — APPROVED** [review]
Looks good to me!

**Review by cmungall — APPROVED** [review]

**Comment by cmungall** [comment]
discuss on next call?

**Comment by matentzn** [comment]
On agenda already!

File: docs/RegistrationChecklist.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Minimum Ontology Registration Request Checklist
+
+The _Minimum Ontology Registration Request Checklist_ (MORRC) is intended to provide a first pass to fairly evaluate ontology registration requests to the OBO foundry. 
+The goal is to formalize what is currently an ad-hoc process, to filter out ontologies that are clearly out of scope for OBO. It is intentionally weaker than full
+conformance, because (a) we want to allow registration of imperfect ontologies (b) partial conformance is easier to quickly and objectively check.
+
+Note this does not change the 2 week rule. Anyone from the OBO community is free to object to the ontology on the basis of any of these checks. 
+However, if no objections are raised in the 2 week period, then the ontology becomes registered. Once registered an ontology is never unregistered.
```
Comment (matentzn) [inline docs/RegistrationChecklist.md:8]: In the Operations call, we discussed that two week the rule is too tight. I suggest changing to "two weeks from next operations call" or "four weeks rule".

File: docs/RegistrationChecklist.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- Principle 8: [Documentation](http://obofoundry.org/principles/fp-008-documented.html)
+  - The ontology SHOULD be conformant with this principle
+- Principle 9: [Documented Plurality of Users](http://obofoundry.org/principles/fp-009-users.html)
+  - The submitter SHOULD state existing users and assumed users
+- Principle 10: [Commitment To Collaboration](http://obofoundry.org/principles/fp-010-collaboration.html)
+  - The submitter SHOULD be committed to collaboration
+- Principle 11: [Locus of Authority](http://obofoundry.org/principles/fp-011-locus-of-authority.html)
+  - The ontology MUST be conformant with this principle
+- Principle 12: [Naming Conventions](http://obofoundry.org/principles/fp-012-naming-conventions.html)
+  - All classes MUST have labels
```
Comment (matentzn) [inline docs/RegistrationChecklist.md:55]: What about obsolete ones?

Comment (nlharris) [reply docs/RegistrationChecklist.md:55]: Maybe we need to add text something like, "To be added to the OBO Foundry, ontologies must meet these requirements. Ontologies that are already listed that do not meet these requirements will be moved to 'inactive' or 'obsolete' status."

File: docs/RegistrationChecklist.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+The checklist is expected to evolve over time; anyone can contribute to the discussion [here](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1116).
+
+## Checklist
+
+- Principle 1: [Open](http://obofoundry.org/principles/fp-001-open.html)
+  - The ontology MUST be conformant with this principle
+- Principle 2: [Common Format](http://obofoundry.org/principles/fp-002-format.html)
+  - The ontology MUST be conformant with this principle
+  - The ontology MUST be parseable in Protégé (or OWLAPI) and be logically consistent with a standard OWL reasoner such as ELK or HermiT
```
Comment (matentzn) [inline docs/RegistrationChecklist.md:26]: Technically the right phrase would be `consistent and coherent`, as we not only want consistency, but also no unsatisfiable named classes (coherency).

File: docs/RegistrationChecklist.md (hunk 2 above)
Comment (matentzn) [reply docs/RegistrationChecklist.md:55]: I meant more, what about obsolete classes; do they have to have labels? I would probably not go into too much depth around when to obsolete an ontology right in this checklist, but we can raise a separate issue!

Comment (nlharris) [reply docs/RegistrationChecklist.md:55]: Ah, sorry, I misinterpreted.

File: docs/RegistrationChecklist.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+- Principle 2: [Common Format](http://obofoundry.org/principles/fp-002-format.html)
+  - The ontology MUST be conformant with this principle
+  - The ontology MUST be parseable in Protégé, ROBOT or OWLAPI and be logically consistent with a standard OWL reasoner such as ELK or HermiT
+- Principle 3: [URI/Identifier Space](http://obofoundry.org/principles/fp-003-uris.html) 
+  - The ontology MUST be conformant with this principle, with the caveat the prefix is not yet registered
+  - The requested namespace MUST be available (not already used by another ontology)
+- Principle 4: [Versioning](http://obofoundry.org/principles/fp-004-versioning.html) 
+  - The ontology MUST have a versionIRI
+- Principle 5: [Scope](http://obofoundry.org/principles/fp-005-delineated-content.html) 
+  - IF there is significant conceptual overlap or matching labels THEN there MUST be computable linkages (one of: skos, obo:xref, owl) which are reasonably comprehensive
```
Comment (cmungall) [inline docs/RegistrationChecklist.md:33]: "IF there is significant conceptual overlap" --> "IF there is significant conceptual overlap to ontologies in OBO"

File: docs/RegistrationChecklist.md (hunk 2 above)
Comment (cmungall) [reply docs/RegistrationChecklist.md:55]: This should read: all non-obsolete classes must have labels

**Comment by jamesaoverton** [comment]
My view is that this is a big step forward, and that we should move forward with it. We can always make refinements later.

**Comment by nlharris** [comment]
Oh, oops, I meant to ask before merging this. Hope it was ok to merge.

**Pull request merged**